typescript-loops/main.ts

```diff
@@ -0,0 +1,103 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen(): Array<number> {
+  const numbers = [];
+  let currentNumber = 1;
+  while (currentNumber < 11) {
+    numbers.push(currentNumber);
+    ++currentNumber;
+  }
+  return numbers;
+}
+
+let results1 = [];
+results1 = getNumbersToTen();
+console.log(results1);
+
+function getEvenNumbersToTwenty(): Array<number> {
+  const evenNumbers = [];
+  let currentNumber = 2;
+
+  while (currentNumber < 21) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+
+const results2 = getEvenNumbersToTwenty();
+console.log('The value of getEvenNumberToTwentyResults:', results2);
+
```

Comment:
The way you called this here is okay though, you just assigned the return to a new variable and then logged the variable.